packages/tupaia-web/src/features/DashboardItem/DashboardInfoHover.tsx

```diff
@@ -57,7 +58,7 @@ export const DashboardInfoHover = ({ infoText }: DashboardInfoHoverProps) => {
   return (
     <Wrapper>
       <MuiInfoIcon />
-      <DashboardInfoHoverText>{infoText}</DashboardInfoHoverText>
+      <DashboardInfoHoverText>{!infoText ? null : infoText}</DashboardInfoHoverText>
```

Comment:
So what I meant here was that `DashboardInfoHoverText` should return null if infoText is unset